src/ansys/aedt/core/modeler/pcb/object_3d_layout.py

```diff
@@ -534,6 +537,78 @@ def enabled(self, status):
             return False
         self._oeditor.EnableComponents(["NAME:Components", self.name], status)
 
+    @property
+    def die_properties(self) -> Optional[Tuple[int, int, str, int]]:
+        """Get die properties from component.
+
+        Returns
+        -------
+        Tuple[int, int, str, int]
+            Tuple of die type (``0`` for None, ``1``, for FlipChip, or ``2`` for WireBond), die orientation (``0`` for
+            Chip Top or ``1`` for Chip Bottom), die height as a string, and a reserved property as an integer.
+        """
+        if self._part_type_id != 4:
+            return None
+        return _get_die_properties(self._get_model_info())
+
+    @property
+    def _has_port_properties(self) -> bool:
```

Comment:
Could you please make this a private method (rename into `__has_port_properties`) instead of a property ? I don't see the point of making it a "protected" property. If there is, please let me know.

Reply:
Per your other comment I will convert both of these properties (new and existing) to methods.

Reply:
Done.